quantus/evaluation.py

```diff
@@ -56,15 +61,18 @@ def evaluate(
         Indicates if progress should be printed to std, or not.
     explain_func_kwargs: dict, optional
         Keyword arguments to be passed to explain_func on call.
```

Comment:
Lets document a bit more concisely that this parameter is not relevant for option 2) of xai_methods